server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessagePreviewGeneratorTest.java

```diff
@@ -64,4 +64,12 @@ public void computeShouldReturnStringIsLimitedTo256Length() throws Exception {
             .hasSize(MessagePreviewGenerator.MAX_PREVIEW_LENGTH)
             .isEqualTo(expected);
     }
+
+    @Test
+    public void computeShouldReturnNormalizeSpaceString() throws Exception {
```

Comment:
Could you have a small test proving it?

I don't like going to read other project documentation, and I don't trust them.... :-)